src/compiler/compile/Component.ts

```diff
@@ -796,6 +796,18 @@ export default class Component {
 					scope = map.get(node);
 				}
 
+				if (node.type === 'ExpressionStatement' && node.expression.type === 'CallExpression') {
```

Comment:
Why only inside `ExpressionStatement`? Function calls happen inside other expressions as well, shouldn't they also be accounted for? Is there a reason `CallExpression` can't be tested by itself?

Reply:
`ExpressionStatement` is the only type with an `expression` key in the `node` object

Reply:
I'm not sure I follow, sorry! Testing for `node.type === 'CallExpression'` is what I meant.

Reply:
On further expression, it appears there are duplicate nodes. There's an `ExpressionStatement` node that contains the `CallExpression` in the `expression` key. There is also another `CallExpression` node. I'm not sure which one is semantically correct but I'll go with CallExpression as it likely encompasses more calls.